src/Components/Event.js

```diff
@@ -1,3 +1,75 @@
-export default function Event() {
-  return;
+export default function Event({
+  event,
+  attendees,
+  updateEventAttendance,
+  setShowAttendees,
+  showAttendees }) {
```

Comment:
Try moving the useState hook for showAttendees into this component instead of passing down the value as a prop!